<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkSummaryWindow</class>
 <widget class="QDialog" name="WorkSummaryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Work Summary</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="layout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>15</number>
     </property>
     <property name="topMargin">
      <number>15</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QComboBox" name="format"/>
     </item>
     <item>
      <widget class="QComboBox" name="period"/>
     </item>
     <item>
      <widget class="QCheckBox" name="view_time_spent">
       <property name="text">
        <string>Include time spent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="view_backlogs">
       <property name="text">
        <string>Include information about backlogs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="work_summary_results"/>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
